app/SmartAlbums/BaseSmartAlbum.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\SmartAlbums;
+
+use App\Contracts\AbstractAlbum;
+use App\Models\Configs;
+use App\Models\Extensions\Thumb;
+
+/**
+ * Class BaseSmartAlbum.
+ *
+ * The common base class for all built-in smart albums which can neither
+ * be created nor deleted, but always exists.
+ * Photos cannot explicitly be added or removed from these albums.
```

Comment:
I see what you mean. That is not what I thought of, but I understand that it might be confusing. What I meant was that these albums are never explicit parents of a photo, but that they contain photos based on certain attributes.

I rephrased the sentence.